src/main/java/Parser.java

```diff
@@ -0,0 +1,153 @@
+import java.io.IOException;
+
+class Parser {
+    /**
+     * Returns the index of the preposition in the String array
+     * If preposition is not found, return 0
+     * @param inputSplit given String array
+     * @return index position or 0
+     */
+    private int getPrepositionPos(String[] inputSplit) {
+        for (int i = 0; i < inputSplit.length; i++) {
+            if (inputSplit[i].charAt(0) == '/') {
+                return i;
+            }
+        }
+        return 0;
+    }
+
```

Comment:
Cool method

Reply:
Thanks! but not sure if you're being sarcastic haha